scripts/ci/run-ci-tests.sh

```diff
@@ -199,6 +199,11 @@ if [ "${STREAM_TEST}" = "1" ]; then
 	exit 0
 fi
 
+# print some useful debug info
+cat /proc/self/status
+ls -la /proc/self/ns
+cat /proc/self/cgroup
+
```

Comment:
If we really want to keep this debug information it should be moved to the `print_env` section above.